tests/unit_tests/pyanaconda_tests/modules/localization/test_module_localization.py

```diff
@@ -171,6 +172,21 @@ def test_collect_requirements(self):
         assert requirements[0].type == "package"
         assert requirements[0].name == "kbd-legacy"
 
+    def test_get_available_translations(self):
+        response = self.localization_interface.AvailableTranslations
+        translations = LanguageData.from_structure_list(response)
+
+        assert len(translations) == 1
+        assert translations[0].english == "English"
+        assert translations[0].lid == "en"
+
+        response = self.localization_interface.GetLanguageLocales("en")
+        locales = LanguageData.from_structure_list(response)
+
+        assert len(locales) == 18
+        assert locales[0].english == "English (United States)"
+        assert locales[0].lid == "en_US.UTF-8"
```

Comment:
Magic number alert :( The "18" feels very fragile. It would be great to include a comment on what the number means and where it comes from (if known). That would help if it starts failing randomly.

Also, is that guaranteed to be always 18, or can a minimized install (say, no `glibc-all-langpacks`) lack some of the data and make it eg. 5?

Is it possible to mock this somehow instead? (It could also turn out to be very uneconomical though...)

Reply:
Indeed good point. Shall I make it len > 1 at this point?

Reply:
Sorry, I just realized that I do know where this comes from - langtable. This means it's static data, always coming together with the library, so the number won't change randomly, only if the package gets updates. I think that's fine then. Maybe a comment that just says it's from langtable to know what broke the test if that happens?

(Note to self: Do not review after 10 PM.)